### ISSUE TITLE
我有一个需要是，要计算二分类的auc，不过Paddle V2好像只支持二维向量，而sigmoid只会输出一个值y，我就需要组合出一个[y, 1-y]这种形式的输出，我该怎么做呢？

### ISSUE THREAD

**Issue by CruiseSun** [PaddlePaddle/Paddle#14442]
_Originally posted by @CruiseSun in https://github.com/PaddlePaddle/Paddle/issues/14428#issuecomment-438977834_

**Comment by CruiseSun** [comment]
请问怎么用log_loss？ https://github.com/PaddlePaddle/Paddle/issues/14428#issuecomment-439039294

**Comment by gavin1332** [comment]
现在log_loss在最新主干上可用了，不过还没有加入到release版本，未来1.2版本应该可以用。
在log_loss不可用的情况下，可以考虑用softmax接cross_entropy做二分类任务，样例如下：

    if hasattr(fluid.layers, "log_loss"):
        mul = fluid.layers.elementwise_mul(user_net, news_net)
        dot = fluid.layers.reduce_sum(input=mul, dim=1, keep_dim=True)
        dot = fluid.layers.sigmoid(dot, inplace=True)
        cost = fluid.layers.log_loss(input=dot, label=label_float, epsilon=1e-5)
        avg_cost = fluid.layers.mean(x=cost)
    else:
        user_net = fluid.layers.relu(user_net, inplace=True)
        news_net = fluid.layers.relu(news_net, inplace=True)
        user_news_concat = fluid.layers.concat([user_net, news_net], axis=1)
        user_news_fc = fluid.layers.fc(
            name="fc_with_softmax",
            input=user_news_concat,
            size=2,
            act="softmax",
            param_attr=fluid.ParamAttr(learning_rate=lr_x))
        cost = fluid.layers.cross_entropy(input=user_news_fc, label=label_data)    
        avg_cost = fluid.layers.mean(x=cost)
